<launch>
    <param name="adns_robot/wheel_base" type="double" value="0.3048" />
    <param name="adns_robot/wheel_radius" type="double" value="0.076" />

    <node pkg="adns_robot_core" type="differential_drive" name="differential_drive" ns="adns_robot" />
    <node pkg="image_proc" type="image_proc" name="image_proc" ns="adns_robot/camera/color">
        <param name="queue_size" type="int" value="1" />
    </node>
</launch>